src/index.ts

```diff
@@ -343,17 +294,33 @@ function launchMetals(
 
     client.onRequest(MetalsInputBox.type, async (options: InputBoxOptions) => {
       const response = await workspace.callAsync<string>("input", [
-        `${options.prompt} `,
-        options.value
+        `${options.prompt} --> `,
```

Comment:
Some times the `InputBoxOptions` prompt that we get has a `?` and other times it has nothing. Just to make it at least seem like there is a break between the prompt and where the user should put their text I've chose a ` --> `. I also thought of doing a ` : `, but that looked a bit odd when I tried with `?:`.

Reply:
Just asked a couple other people, and they all thought `:` was a better option, so I added that in instead.